src/graphics/window.rs

```diff
@@ -20,6 +22,8 @@ pub struct Window {
     width: f32,
     height: f32,
     is_fullscreen: bool,
+    cursor_icon: winit::window::CursorIcon,
+    cursor_visible: bool,
```

Comment:
We could probably use an `Option` to encode both things.

Reply:
Addressed in 66828daceee95b73aa10a45c2d279fe7600e3660.